.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 rvm:
- - 1.9.3
  - 2.1.8
  - 2.2.4
+ - 2.5.1
```

Comment:
@viraptor we don't need to support 1.9 anymore do we?

Reply:
I don't believe so. But 2.4 could be nice to test against as well.

Reply:
I'll add it

Reply:
Any reason not to test on the latest patch releases (2.4.5 and 2.5.3)?

It'd be nice to add 2.6.1 as well.